src/main/java/com/arcbees/website/client/application/bees/BeesView.ui.xml

```diff
@@ -313,24 +339,36 @@
             <div class="{res.style.stripe}">
                 <div class="{res.grid.row} {res.style.clearfix} {res.style.wrapper}">
                     <div class="{res.grid.col} {res.grid.col_full}">
-                        <h3>Software engineer</h3>
-                        <a href="#" class="{res.style.btn}">How can you be the best programmer you can be</a>
+                        <h3>
+                            <ui:msg description="Job applications - Software engineer">Software engineer</ui:msg>
+                        </h3>
+                        <a href="mailto:candidates@arcbees.com" class="{res.style.btn}">
+                            <ui:msg description="Job applications - Software engineer - Btn">How can you be the best programmer you can be</ui:msg>
+                        </a>
                     </div>
                 </div>
             </div>
             <div class="{res.style.stripe}">
                 <div class="{res.grid.row} {res.style.clearfix} {res.style.wrapper}">
                     <div class="{res.grid.col} {res.grid.col_full}">
-                        <h3>Web developer</h3>
-                        <a href="#" class="{res.style.btn}">Awesome CSS Master</a>
+                        <h3>
+                            <ui:msg description="Job applications - Web developer">Web developer</ui:msg>
+                        </h3>
+                        <a href="mailto:candidates@arcbees.com" class="{res.style.btn}">
+                            <ui:msg description="Job applications - Web developer - Btn">Awesome CSS Master</ui:msg>
+                        </a>
                     </div>
                 </div>
             </div>
             <div class="{res.style.stripe}">
                 <div class="{res.grid.row} {res.style.clearfix} {res.style.wrapper}">
                     <div class="{res.grid.col} {res.grid.col_full}">
-                        <h3>Designer slave</h3>
-                        <a href="#" class="{res.style.btn}">We hope you like to suffer</a>
+                        <h3>
+                            <ui:msg description="Job applications - Graphic designer">Graphic designer</ui:msg>
+                        </h3>
+                        <a href="mailto:candidates@arcbees.com" class="{res.style.btn}">
+                            <ui:msg description="Job applications - Web developer - Btn">We hope you like to suffer</ui:msg>
```

Comment:
As we removed "slave" in the title, that line should be changed to. It's not a good idea to tell people that they will suffer when what we want to do is to look like a fun place and drive talented people to us. It's hard to read sarcasm and the previous jobs are about "best programmer" and "awesome css master" so this one doesn't fit.